### ISSUE TITLE
Fifth metatarsal apophysitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11192]
**Preferred term label:**
Fifth metatarsal apophysitis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Oblique foot x-ray showing apophyseal irregularity (enlarged apophysis, disordered ossification, widened chondro-osseous junction) at the base of the 5th (fifth) metatarsal; Iselin disease

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Iselin's disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001344
- label: Fifth metatarsal apophysitis
- parents: [Abnormality of the fifth metatarsal bone (HP:0008089)](https://hpo.jax.org/app/browse/term/HP:0008089)
- pmid: PMID:27546974
- def: Iselin's disease is a traction apophysitis of base of fifth metatarsal. The apophysis is within the peroneus brevis tendon insertion site and appears radiographically at about age 10 in female and age 12 in male and usually fuses by age 11 in female and by age 14 in male. Repetitive microtrauma from the peroneus brevis tendon, as occurs in sports activities is the cause of traction apophysitis. It can be ascertained by oblique foot radiographs showing apophyseal irregularity (enlarged apophysis, disordered ossification, widened chondro-osseous junction) at the base of the fifth metatarsal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11192
- synonyms: Iselin disease(NONE)
